tests/NHSD.BuyingCatalogue.Testing.Data/SqlResources/ReferenceData.sql

```diff
@@ -4,7 +4,8 @@ INSERT INTO [dbo].[CapabilityStatus] ([Id] ,[Name]) VALUES (1,'Effective')
 
 INSERT INTO [dbo].[CapabilityCategory] ([Id] ,[Name]) VALUES (0,'Undefined')
 
-INSERT INTO [dbo].[SolutionCapabilityStatus] ([Id], [Name], [Pass]) VALUES (1, 'Passed', 0);
+INSERT INTO [dbo].[SolutionCapabilityStatus] ([Id], [Name], [Pass]) VALUES (1, 'Passed', 1);
```

Comment:
This looks a lot better. In the actual data model we have the following statuses

(1, 'Passed - Full', 1);
(2, 'Passed - Partial', 1);
(3, 'Failed', 0);

Should we be using these? I'm not sure it matters at this stage.

Reply:
Hmm. Part of me says to keep the data consistent. However, I think in this case, it's better to have it subtly differnt, so it forces you to use the passed flag, rather than depending on the status ids